README.md

```diff
@@ -61,14 +61,6 @@ Add it to your Gemfile and `bundle install`:
 gem "rbexy"
 ```
 
-_From 1.0 onward, we only support Rails 6. If you're using Rails 5, use the 0.x releases._
-
-In `config/application.rb`:
-
-```ruby
-require "rbexy/rails/engine"
-```
```

Comment:
Why is this removed?

Reply:
Whoops. This is suppose to be in a different PR I didn't push yet.